_includes/configuration/language.md

```diff
@@ -28,83 +28,83 @@ tinymce.init({
-| Portuguese (Portugal)         | pt_PT   |
-| Romanian                      | ro      |
-| Russian                       | ru      |
-| Serbian                       | sr      |
-| Sinhala (Sri Lanka)           | si_LK   |
-| Slovak                        | sk      |
-| Slovenian (Slovenia)          | sl_SI   |
-| Spanish                       | es      |
-| Spanish (Mexico)              | es_MX   |
-| Swedish (Sweden)              | sv_SE   |
-| Tajik                         | tg      |
-| Tamil                         | ta      |
-| Tamil (India)                 | ta_IN   |
-| Tatar                         | tt      |
-| Thai (Thailand)               | th_TH   |
-| Turkish                       | tr      |
-| Turkish (Turkey)              | tr_TR   |
-| Uighur                        | ug      |
-| Ukrainian                     | uk      |
-| Ukrainian (Ukraine)           | uk_UA   |
-| Vietnamese                    | vi      |
-| Vietnamese (Viet Nam)         | vi_VN   |
-| Welsh                         | cy      |
+| Language                      |  Code   | {{ site.cloudname }} /{{ site.newline }}Enterprise |
+| :-----------------------------| :-----: | :------------------------------------------------: |
+| Arabic                        | ar      | {{ site.tick }}                                    |
+| Arabic (Saudi Arabia)         | ar_SA   |                                                    |
+| Armenian                      | hy      |                                                    |
+| Azerbaijani                   | az      |                                                    |
+| Basque                        | eu      | {{ site.tick }}                                    |
+| Belarusian                    | be      |                                                    |
+| Bengali (Bangladesh)          | bn_BD   |                                                    |
+| Bosnian                       | bs      |                                                    |
+| Bulgarian (Bulgaria)          | bg_BG   | {{ site.tick }}                                    |
+| Catalan                       | ca      | {{ site.tick }}                                    |
+| Chinese (China)               | zh_CN   | {{ site.tick }}                                    |
+| Chinese (Taiwan)              | zh_TW   | {{ site.tick }}                                    |
+| Croatian                      | hr      | {{ site.tick }}                                    |
+| Czech                         | cs      | {{ site.tick }}                                    |
+| Danish                        | da      | {{ site.tick }}                                    |
+| Divehi                        | dv      |                                                    |
+| Dutch                         | nl      | {{ site.tick }}                                    |
+| English (Canada)              | en_CA   |                                                    |
```

Comment:
Should we add en_us for completeness?

Reply:
Maybe, though this is really talking about language packs you can download. `en_US` would be the default.